cri/v1alpha2/cri.go

```diff
@@ -244,34 +243,50 @@ func (c *CriManager) RunPodSandbox(ctx context.Context, r *runtime.RunPodSandbox
 	}
 
 	// Step 2: Create the sandbox container.
+
```

Comment:
delete this blank line ?

Reply:
This comment `Step2` represents it's a section down here , and should be separated from the comments below by a blank line. So I suggest keeping it. ^_^